dll/directx/wine/wined3d/context.c

```diff
@@ -5708,7 +5708,7 @@ void context_draw_shaded_quad(struct wined3d_context *context, struct wined3d_te
         GL_EXTCALL(glBufferData(GL_ARRAY_BUFFER, sizeof(quad), quad, GL_STREAM_DRAW));
         GL_EXTCALL(glVertexAttribPointer(0, 2, GL_FLOAT, FALSE, sizeof(*quad), NULL));
         GL_EXTCALL(glVertexAttribPointer(1, 3, GL_FLOAT, FALSE, sizeof(*quad),
-                (void *)FIELD_OFFSET(struct blit_vertex, texcoord)));
+                UlongToPtr(FIELD_OFFSET(struct blit_vertex, texcoord))));
```

Comment:
Wine code. Disable the warning?
Same for clusapi, crypt32, mshtml, msi, msrle32, gdiplus_winetest

Reply:
Sending patch upstream too?

Reply:
I made them non-fatal for now.